docs/specs/xref.yml

```diff
@@ -200,29 +202,31 @@ outputs:
     {"!references":null}
 ---
 # Resolve xref in SDP
-inputs:
-  docfx.yml:
-  docs/a.json: |
-    {
-      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
-      "uid": "a",
-      "name": "test",
-      "fullName": "test.test",
-      "description": "a.description"
-    }
-  docs/b.json: | 
-    {
-      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
-      "xref": "a"
-    }
+repos:
+  https://docs.com/test-xref-sdp#live:
+    - files:
+        docfx.yml:
+        docs/a.json: |
+          {
+            "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+            "uid": "a",
+            "name": "test",
+            "fullName": "test.test",
+            "description": "a.description"
+          }
+        docs/b.json: | 
+          {
+            "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+            "xref": "a"
+          }
 outputs:
   docs/a.json:
   docs/b.json: | 
     {
       "content": {
         "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
         "xref": {
-            "href": "https://docs.com/docs/a.json",
+            "href": "/docs/a.json",
```

Comment:
why it's removed host?

Reply:
internal defined UID should be resolved to relative path